front/app/containers/Admin/projects/project/timeline/edit.tsx

```diff
@@ -228,20 +230,22 @@ const AdminProjectTimelineEdit = () => {
       setProcessing(true);
       if (!isEmpty(attributeDiff)) {
         if (phase) {
-          setRedirectAfterSave(false);
```

Comment:
I noticed another bug with the redirect and the `handleSaveResponse` function didn't use the value as updated here (I'm assuming since it didn't rerender?). Instead I chose to remove the setRedirectAfterSave and just pass it in directly (and added typing to the function for the response).